README.md

```diff
@@ -79,5 +119,15 @@ If you added your libraries to a directory in your `PATH`, no extra steps are ne
     `LD_LIBRARY_PATH` environment variable. Note that this directory does not have to be the same added to
     `LIBRARY_PATH` in the compilation step.
 
+### Execution (static libraries)
+
+#### macOS-only (target iOS)
```

Comment:
Naming the section "macOS-only" may be confusing, specially if a section for targeting macOS is added later on. It may be better to just name it "iOS-only". As far as I'm aware, that aready implies that you're compiling from macOS, right?